### ISSUE TITLE
Why train using one dataset (VGGFace2) but validate using another dataset(LFW)?

### ISSUE THREAD

**Issue by gp-wang** [davidsandberg/facenet#914]
I have a question related to the training script in the wiki:

From a general machine learning point of view, isn't it true that we create a validation dataset by extracting some portion (e.g. 1/10) from the training set? 

But why it is recommended here that to train using VGGFace dataset but validate using lfw dataset? Aren't they two unrelated dataset? 

If I am to train for my own dataset, say Microsoft-1M-Celebrity, how I am going to write the training script? Shall I keep using LFW as the validation set? Or shall I extract a validation set from the training set?

Appreciate your answer

```
python src/train_softmax.py \
--logs_base_dir ~/logs/facenet/ \
--models_base_dir ~/models/facenet/ \
--data_dir ~/datasets/vggface2/vggface2_train_182/ \
--image_size 160 \
--model_def models.inception_resnet_v1 \
--lfw_dir ~/datasets/lfw/lfw_mtcnnalign_160/ \
--optimizer ADAM \
--learning_rate -1 \
--max_nrof_epochs 500 \
--batch_size 90 \
--keep_probability 0.4 \
--random_flip \
--use_fixed_image_standardization \
--learning_rate_schedule_file data/learning_rate_schedule_classifier_vggface2.txt \
--weight_decay 5e-4 \
--embedding_size 512 \
--lfw_distance_metric 1 \
--lfw_use_flipped_images \
--lfw_subtract_mean \
--validation_set_split_ratio 0.01 \
--validate_every_n_epochs 5
```

**Comment by Salary-only-17k** [comment]
i have the same trouble. i found that the 'pair.txt' file donot contain the VGGface set labels, and lfw dataset is not in the 'pair.txt' file.  i saw 'how to generate pair.txt' issue , and the validate meth create new pair.txt?

**Comment by hsm4703** [comment]
this quesion have fix it  i think the model just learning common features  like nose 、eye、mouse  ，use  model weights and bias to evaulation lfw pairs.txt. l think use model to pair.txt Absolute path and name to calculation pair.txt file everyone embedding and compare two pepple  128 embedding  distance . i don't know my idea right or false?
